### PR TITLE
tests: allow to select a suite when running autopkgtests

### PR DESCRIPTION
**Pull request by elopio**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----
This PR needs to be split in two, because I first need this tiny change in the PPA in order to use it in the travis daily. I'll propose the second part once I get this change into the daily PPA.

### CODE REVIEW

**Comment by kalikiana** [comment]
@elopio I'm slightly confused... is this part of the split, or are you still going to close this and propose separate PRs?

**Comment by elopio** [comment]
@kalikiana I'm not going to close this. I need this to land, to then update the autopkgtests in the PPA, and then I can propose the PR that executes those tests.

File: debian/tests/integrationtests [hunk 1]
```diff
@@ -4,7 +4,16 @@ set -e
 
 echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers.d/autopkgtest
 
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests/store"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 TEST_STORE=fake ADT_TEST=1 python3 -m unittest discover -b -v -s integration_tests/plugins"
-su ubuntu -c "SNAPCRAFT_FROM_INSTALLED=1 SNAPCRAFT_SLOW_TESTS=1 TEST_STORE=fake ADT_TEST=1 PATH=/snap/bin:$PATH python3 -m unittest discover -b -v -s integration_tests/snapd"
+if [ -z "$SNAPCRAFT_AUTOPKGTEST_SUITES" ]; then
+    suites="integration_tests integration_tests/store integration_tests/plugins integration_tests/snapd"
```
Comment (kyrofa) [inline debian/tests/integrationtests]: Since you're running a `discover` below, are you sure you want the parent dir (`integration_tests`) included here?

Comment (kyrofa) [reply debian/tests/integrationtests]: Scratch this comment-- these dirs aren't packages (no `__init__.py`.

**Comment by kalikiana** [comment]
`autopkgtest [00:43:00]: ERROR: git-source https://github.com/snapcore/snapcraft.git#refs/pull/1630/head has no test matching --test-name integration` on xenial-amd64. Not sure what that means...

**Pull request merged**